test/built-ins/Object/entries/exception-during-enumeration.js

```diff
@@ -2,8 +2,8 @@
 // This code is governed by the BSD license found in the LICENSE file.
 
 /*---
+id: sec-object.entries
```

Comment:
We should wait until the proposal gets merged into the draft to use the ID. This one doesn't match what's [currently in the proposal](https://github.com/tc39/proposal-object-values-entries/blob/master/index.html#L63) anyway.